src/Features/Core/Portable/EmbeddedLanguages/RegularExpressions/RegexCharClass.cs

```diff
@@ -201,206 +189,49 @@ internal static class RegexCharClass
         public static bool IsEscapeCategory(string value)
             => EscapeCategories.ContainsKey(value);
 
-        public static bool IsWordChar(VirtualChar r)
+        public static bool IsBoundaryWordChar(VirtualChar r)
         {
```

Comment:
My preference would be to keep the naming the same as in the runtime side. But I suppose the difference is minimal enough that someone can figure this out.

Reply:
Weird. I followed https://github.com/dotnet/runtime/blob/5b5bd46c03c86f8545f2c4c8628ac25d875210fe/src/libraries/System.Text.RegularExpressions/src/System/Text/RegularExpressions/RegexCharClass.cs (the link at the top), and it had IsWordChar. Can you update the link?

Reply:
> Weird. I followed https://github.com/dotnet/runtime/blob/5b5bd46c03c86f8545f2c4c8628ac25d875210fe/src/libraries/System.Text.RegularExpressions/src/System/Text/RegularExpressions/RegexCharClass.cs (the link at the top), and it had IsWordChar. Can you update the link?

It has both:
https://github.com/dotnet/runtime/blob/5b5bd46c03c86f8545f2c4c8628ac25d875210fe/src/libraries/System.Text.RegularExpressions/src/System/Text/RegularExpressions/RegexCharClass.cs#L1100
https://github.com/dotnet/runtime/blob/5b5bd46c03c86f8545f2c4c8628ac25d875210fe/src/libraries/System.Text.RegularExpressions/src/System/Text/RegularExpressions/RegexCharClass.cs#L1133